sdk/communication/AzureCommunicationChat/Source/Generated/Chat.swift

```diff
@@ -17,7 +17,7 @@ import Foundation
 // swiftlint:disable type_body_length
 
 public final class Chat {
-    public let client: AzureCommunicationChatClient
+    internal let client: AzureCommunicationChatClient
```

Comment:
Should AzureCommunicationChatClient also be internal?

Reply:
Yup, all tests passed. We can't set AzureCommunicationChatClient to internal, because we pick up AzureCommunicationChatClient.apiVersion in AzureCommunicationChatClientOptions, where we call AzureCommunicationChatClientOptions as an input param in ChatThreadClient init()